pipeline_noted.ipynb

```diff
@@ -507,7 +622,10 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Next we drop along `frame` dimension `where` the frame is absolutely dark (the `sum` of all pixel values are zero). Note this step cannot be lazy, i.e. the actual `sum` computation has to be carried out here, so it might not be very efficient if `in_memory=False`. Feel free to skip this step if you are sure there is no such black frames in your data."
+    "Next we drop along `frame` dimension `where` the frame is absolutely dark (the `sum` of all pixel values are zero). Note this step cannot be lazy, i.e. the actual `sum` computation has to be carried out here, so it might not be very efficient if `in_memory=False`. \n",
+    "<div class=\"alert alert-success\">                                                               \n",
+    "Feel free to skip this step if you are sure there is no such black frames in your data.\n",
```

Comment:
I think you're over-doing this... it's gonna feel so out-of-place with a box containing single sentence that flows well with the context